src/Components/Todos/TodoInput/TodoInput.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class TodoInput extends React.Component {
+  state = {
+    input: '',
+  };
+
+  handleAddInputValue = (event) => {
+    const { input } = this.state;
+
+    if (event.key === 'Enter' && input) {
+      event.preventDefault();
+      this.props.addTodo(input);
+      this.setState({ input: '' });
+    }
+  }
+
+  handleInputChange = (event) => {
+    const controlledInput = (event.target.value).trim();
+
+    this.setState({
+      input: controlledInput,
+    });
```

Comment:
немного поменяем местами и сделаем деструктуризацию
```suggestion
    const { value } = event.target;

    this.setState({
      input: value.trim(),
    });
```